include/localization.js

```diff
@@ -472,7 +472,6 @@ module.exports = function LocalizationModule(pb) {
      * @param {Function} cb
      */
     Localization.init = function(cb) {
-        var supportedLocales = [];
```

Comment:
That is my guess or commits got lost on one end but are being pushed back by your fork.  Not sure but seems to be all there between your push and the branch we have.